src/sentry/static/sentry/app/views/events/eventsChart.jsx

```diff
@@ -86,7 +86,7 @@ class EventsAreaChart extends React.Component {
         grid={{
           left: '24px',
           right: '24px',
-          top: '24px',
+          top: '32px',
```

Comment:
@doralchan this gives clearance for the legend.

<img width="1181" alt="Screen Shot 2020-03-03 at 10 07 34 PM" src="https://user-images.githubusercontent.com/139499/75841307-67bd6180-5d9b-11ea-882b-3aad678753ea.png">